services/common/src/interfaces/report/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./mineReportDefinition.interface";
```

Comment:
I guess- I called this directory _reports_. Doesn't matter to me, I am fine with either, but you'll probably want to pull now that it's merged and make it the same.